v19.2/vectorized-execution.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Vectorized Query Execution
+summary: The CockroachDB vectorized SQL query execution engine processes query plans using a column-oriented model to improve performance.
+toc: true
+---
+
+ <span class="version-tag">New in v19.2:</span> CockroachDB supports column-oriented ("vectorized") query execution on all [supported SQL operations](https://www.cockroachlabs.com/docs/stable/sql-feature-support.html).
```

Comment:
Same comments as above re: "I think the text "supports vectorized .. on all supported SQL operations" is slightly misleading from a user POV, since the "Vectorized Query Execution" page then goes on to list all sorts of contexts in which it does not run stuff in vectorized like "streaming operations" etc etc.  It also reads phrasing-wise as a little tautological, "we support all the supported things". Not sure how to suggest a rephrasing in a way that makes sense, since I don't have much technical context here.  Happy to talk offline also if that would be useful."  Maybe something like "the vectorized engine is enabled for many SQL queries that would benefit from columnarized execution", if that is in fact the case?